### PR TITLE
Add toleration command

### PR DESCRIPTION
**Pull request by Kasama**
Add support for Pod tolerations in the command line

```
Valid commands for toleration:

toleration:list        list toleration for an app
toleration:set         set toleration for an app
toleration:unset       unset toleration for an app
```

Also part of this feature:
topfreegames/controller#6
topfreegames/controller-sdk-go#4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,13 +15,12 @@ ENV CGO_ENABLED=0
 ADD https://codecov.io/bash /usr/local/bin/codecov
 RUN chmod +x /usr/local/bin/codecov
 
-COPY glide.yaml /go/src/github.com/deis/workflow-cli/
-COPY glide.lock /go/src/github.com/deis/workflow-cli/
+COPY . /go/src/github.com/deis/workflow-cli
```
Comment (cscatolini) [inline Dockerfile]: We should copy only `Gopck.lock` and `Gopck.toml`

**Review by cscatolini — COMMENTED** [review]
Good job! 

I have only an small comment about the Dockerfile before we're good to go.

File: Dockerfile (hunk 1 above)
Comment (Kasama) [reply Dockerfile]: I've tried copying just the `Gopkg.{lock,toml}` files, but `dep ensure` complains that `no dirs contained any Go code`. That's why I'm copying the whole directory, together with the Go files

Comment (Kasama) [reply Dockerfile]: although I've just found out that there is a `--vendor-only` option to `dep`, to make it so it doesn't try to resolve the dependencies in go code. That should work

**Review by cscatolini — APPROVED** [review]

**Pull request merged**